README.md

```diff
@@ -1,2 +1,103 @@
-# module-ballerinai-observe
-Observability components related to the internal executions in Ballerina
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinai-observe.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/http.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fhttp)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinai-observe/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinai-observe)
+
+This module contains internal configurations and initializations for Ballerina Observability. Ballerina supports Observability out of the box. You can use [module-ballerina-observe](https://github.com/ballerina-platform/module-ballerina-observe) in your ballerina project and enable observability features.
+
+By default, observability is not included in the executable created by Ballerina. It can be added by using the `--observability-included` build flag or by adding the following section to the `Ballerina.toml` file.
+
+
+
+## Issues and projects
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+This repository only contains the source code for the package.
+
+## Build from the source
+
+### Set Up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+
+    * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+    * [OpenJDK](https://adoptopenjdk.net/)
+
+      > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+2. Export GitHub Personal access token with read package permissions as follows,
+
+        export packageUser=<Username>
+        export packagePAT=<Personal access token>
+
+3. Download and install [Docker](https://www.docker.com/).
```

Comment:
Do we need docker ? 🤔

Reply:
@keizer619

Reply:
we dont need docker for this module.

Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/34/commits/b7bfca5c03381e9ccfdfd7ae6a6a4c64467271a4